src/main/java/homework_4/customAnnotation/Coffee.java

```diff
@@ -0,0 +1,39 @@
+package homework_4.customAnnotation;
+
+import lombok.Data;
+import lombok.SneakyThrows;
```

Comment:
opt: as for me, it's not looks clear when you using code generating in your custom annotation
`SneakyThrows` looking strange here, do you really need it?

Reply:
Thank you so much!